App_Config/Include/SharedSource.RedirectModule.config

```diff
@@ -1,22 +1,29 @@
-<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/">
+<!-- 
+  This file was extended with Sitecore 9.0 configuration roles to natively support Sitecore 9.0. 
+  Previous versions of Sitecore such as 8.0, 8.1, 8.2 will ignore these configuration roles notations 
+  so there won't be any difference there, unless Sitecore-Configuration-Roles is installed (8.1.3+) which 
+  supports these notations same way as Sitecore 9.0 OOB.
+-->
+<configuration xmlns:patch="http://www.sitecore.net/xmlconfig/" xmlns:role="http://www.sitecore.net/xmlconfig/role/">
   <sitecore>
     <pipelines>
       <httpRequestBegin>
         <processor type="SharedSource.RedirectModule.Processors.RedirectProcessor,SharedSource.RedirectModule" patch:after="processor[@type='Sitecore.Pipelines.HttpRequest.ItemResolver, Sitecore.Kernel']"/>
       </httpRequestBegin>
 
-      <getContentEditorWarnings>
+      <getContentEditorWarnings role:require="Standalone or ContentManagement">
```

Comment:
This pipeline is only needed on CM instance, as well as commands and events below.